corehq/apps/sms/static/sms/js/compose.js

```diff
@@ -1,4 +1,10 @@
-hqDefine("sms/js/compose", function() {
+hqDefine("sms/js/compose",[
+    "jquery",
+    "hqwebapp/js/initial_page_data",
+    'jquery-ui/ui/sortable',
+    "hqwebapp/js/bootstrap-multi-typeahead",
+    "bootstrap3-typeahead/bootstrap3-typeahead.min",
```

Comment:
Once `hqwebapp/js/bootstrap-multi-typeahead` is migrated, this can be a dependency of that file rather than this one.